views/subs/aboutTabs.erb

```diff
@@ -0,0 +1,53 @@
+<!-- Application -->
+<!--//about, projects, rationale, ambition, learning_experience, how_did_you_hear-->
+<% def ShortWords(theString, maxLength)
+	if theString.length > maxLength
+		outstring = theString[0..maxLength-3]+"..."
+		return outstring
+	else
+		return theString
+	end
+end%>
+
+<%maxLength = 8%>
+
+<% appFields = [
+	['about','Please tell us a bit about yourself'],
+	['projects','Please tell us about a programming project that you worked on and what your role was'],
+	['rationale',"Why do you want to apply for this program?"],
+	['ambition',"Where do you see your career in 1 year's time?"],
+	['learning_experience',"What do you think makes for a positive learning and training experience?"],
+	['how_did_you_hear',"How did you hear about us?"]
+	] %>
```

Comment:
I see where you're going with this- how easy is your code to change? What would happen if we added a new field for instance? Might there be ways of altering the code so it's more change resistant?